src/html/environment.html

```diff
@@ -179,7 +179,7 @@
 			if (!unitURL.endsWith("/")) {
 				unitURL = unitURL + "/";
 			}
-			cellUrl = unitURL + cellName + "/";
+			unitURL = unitURL + cellName + "/";
```

Comment:
Why do you assign value to unitURL but never use it? Shouldn't it be cellURL?

Reply:
cellURL is correct.